libsrc/cx16/crt0.s

```diff
@@ -62,6 +62,7 @@ L2:     lda     zpsave,x
         stx     VIA1::PRA2      ; Restore former RAM bank
         lda     VIA1::PRB
         and     #<~$07
+        ora     #$02
```

Comment:
The BASIC interpreter now will be in bank four.

Reply:
Not sure how I made that mistake. Fixed it in the force push.

Reply:
BASIC might have migrated from bank zero to two to four.